commcare_connect/opportunity/views.py

```diff
@@ -1600,7 +1600,7 @@ def user_visit_details(request, org_slug, opp_id, pk):
     verification_flags_config = opportunity.opportunityverificationflags
     deliver_unit_flags_config = DeliverUnitFlagRules.objects.filter(
         opportunity=opportunity, deliver_unit=user_visit.deliver_unit
-    )
+    ).first()
```

Comment:
what is this for? why would there be more than one, and if there is, is it safe to ignore the second?